src/plot.py

```
@@ -0,0 +1,147 @@
+            pt_change=html("Percent point change *"),
+        )
+        .tab_header(
+            title=md(title),
+        )
+        .tab_source_note(
+            source_note=html(
+                f"""
+                Updated as of {cur.strftime("%-d %b %Y")}. Referencing the latest data
+                that overlaps with the current date. See further details from the
+                <a href='https://www.ipcinfo.org/ipc-country-analysis/en/'>IPC website
+                </a>.<br><br> * A value of 0 may also indicate that no comparison
+                could be made due to large differences in analyzed populations.
+                """
+            )
+        )
+    )
+
+
+def summary_table(df, changes_df=None):
+    cur = pd.Timestamp.now()
+
+    df_display = df.copy()
+
+    # Add direction column
+    if changes_df is not None:
+        severity_order = {"low": 0, "medium": 1, "high": 2, "very high": 3}
+
+        for idx in changes_df.index:
+            # Get first alert column that changed
+            alert_cols = ["max_alert_level", "alert_level_hs", "alert_level_ipc"]
+            for col in alert_cols:
+                if col in changes_df.columns:
+                    old_val = changes_df.loc[idx, (col, "other")]
+                    new_val = changes_df.loc[idx, (col, "self")]
+                    try:
+                        if severity_order[old_val] < severity_order[new_val]:
+                            df_display.loc[idx, "country"] += (
+                                ' <span style="color: red;">▲</span>'
+                            )
+                        elif severity_order[old_val] > severity_order[new_val]:
+                            df_display.loc[idx, "country"] += (
+                                ' <span style="color: green;">▼</span>'
+                            )
+                    except Exception:
+                        break
+                    break
```

do we want a silent error here?